lib/lens/compression.rb

```diff
@@ -0,0 +1,97 @@
+require 'bson'
+require 'beefcake'
+
+module Lens
+  module Compression
+    module Gzip
+      def self.compress(data)
+        GzipUtil.gzip(data)
+      end
+
+      def self.headers
+        {'Content-Encoding' =>'gzip'}
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.